explorer/src/providers/stats/solanaDashboardInfo.tsx

```diff
@@ -0,0 +1,96 @@
+import { PerfSample } from "@solana/web3.js";
+import { ClusterStatsStatus } from "./solanaClusterStats";
+
+export type DashboardInfo = {
+  status: ClusterStatsStatus;
+  avgBlockTime_1h: number;
+  avgBlockTime_1min: number;
+  epochInfo: {
+    absoluteSlot: number;
+    blockHeight: number;
+    epoch: number;
+    slotIndex: number;
+    slotsInEpoch: number;
+  };
+};
+
+export enum DashboardInfoActionType {
+  SetPerfSamples,
+  SetEpochInfo,
+  SetError,
+  Reset,
+}
+
+export type DashboardInfoActionSetPerfSamples = {
+  type: DashboardInfoActionType.SetPerfSamples;
+  data: PerfSample[];
+};
+
+export type DashboardInfoActionSetEpochInfo = {
+  type: DashboardInfoActionType.SetEpochInfo;
+  data: any;
+};
+
+export type DashboardInfoActionReset = {
+  type: DashboardInfoActionType.Reset;
+  data: DashboardInfo;
+};
+
+export type DashboardInfoActionSetError = {
+  type: DashboardInfoActionType.SetError;
+  data: string;
+};
+
+export type DashboardInfoAction =
+  | DashboardInfoActionSetPerfSamples
+  | DashboardInfoActionSetEpochInfo
+  | DashboardInfoActionReset
+  | DashboardInfoActionSetError;
+
+export function dashboardInfoReducer(
+  state: DashboardInfo,
+  action: DashboardInfoAction
+) {
+  const status =
+    state.avgBlockTime_1h !== 0 && state.epochInfo.absoluteSlot !== 0
+      ? ClusterStatsStatus.Ready
+      : ClusterStatsStatus.Loading;
+
+  switch (action.type) {
+    case DashboardInfoActionType.SetPerfSamples:
+      const samples = action.data.map((sample) => {
+        return sample.samplePeriodSecs / sample.numSlots;
+      });
```

Comment:
The average block time stat used to filter out skipped slots. This looks like it's measuring the average time elapsed per slot rather than average block time. Slot time is useful for measuring the progress of an epoch but I think it needs to be coupled with some indication of the skipped slot percentage to give the full picture of how long it takes for a transaction to land.

Reply:
This is interesting. I guess I didn't know how average block time was computed. Since we are using the performance samples in this manner, would it make sense for us to filter out the skipped slots on the backend when we write PerfSamples to the blockstore?

Reply:
Just noticing "Epoch time remaining" is off from the Solana Beach version.

Reply:
Oh interesting, Solana Beach now shows "Slot Time" instead of "Block Time". Seems fine for now to do the same in this PR.

> would it make sense for us to filter out the skipped slots on the backend when we write PerfSamples to the blockstore?

Maybe we can add that later. Slot time seems fine for now.

> Just noticing "Epoch time remaining" is off from the Solana Beach version.

On this branch?

Reply:
Yeah, it's happening in this branch. I will investigate.

Reply:
@oJshua are you still looking into this?

Reply:
Yes, so the hourly average block time being provided by Solana Beach is around 10% off from our computed value. I seem to trust our data more. I can monitor when the epoch transition happens and see who is more accurate.

Reply:
Sounds good. I think this PR just needs a change from "Block Time" to "Slot Time" then

Reply:
Thanks! Pushed.